### ISSUE TITLE
Password is shown in clear when added from stdin

### ISSUE THREAD

**Issue by jle64** [Keruspe/GPaste#210]
When adding a password by piping it to gpaste-client stdin, it gets duplicated as a normal entry in addition to the password entry (see the 0 entry that shouldn't be here) :
```
$ gpaste-client delete-history 
$ echo test | gpaste-client add-password test
$ gpaste-client history
0: test
1: [Password] test
```

**Comment by Keruspe** [comment]
Hi,

Which version of GPaste is this?

**Comment by jle64** [comment]
3.24.1

**Comment by Keruspe** [comment]
Oh, I realize that I still haven't released 3.24.2, will do this week.
This issue is already fixed for this release :)

**Comment by jle64** [comment]
Thank you !
